admin_manual/release_notes.rst

```diff
@@ -12,6 +13,61 @@ Release Notes
+
+- **Configurable solution for indistinguishable user display names**
+In the ownCloud sharing dialog users are displayed according to their display name. As users can choose their display name in self-service (can be disabled in config.php) and display names are not unique it is possible that a user can't distinguish sharing results. To cover this case the displayed user identifiers are now configurable. Administrators can configure to additionally display mail addresses or user ids. Please check if this new feature better suits your needs.
+
+- **Added "occ files:scan" repair mode to repair filecache inconsistencies**
+We recommend to use this command when directed to do so in the upgrade process (TODO more info)
+Please check the ownCloud documentation for more information. **LINK (https://github.com/owncloud/documentation/issues/3446)**
+
+- **Detailed mode for "occ security:routes"**
+Administrators can use the output of this command when using a network firewall to check the appropriateness of configured rules or to get assistance when setting up.
+
+- **Added mode of operations to differentiate between single-instance or clustered setup**
+As ownCloud needs to behave differently when operating in a clustered setup we added a config.php option ``'operation.mode' => 'single-instance' or 'clustered-instance'``. 
+Currently the Market App (ownCloud Marketplace integration) does not support clustered setups and can do harm when used for installing or updating apps. The new config setting prevents this and other actions that are undesired in cluster mode.
+**Therefore when operating in a clustered setup it is mandatory to set this option.**
+Please check the ownCloud documentation for more information. **LINK (https://github.com/owncloud/documentation/pull/3506)**
+
+- **Added occ dav:cleanup-chunks command to clean up expired uploads**
+When file uploads are interrupted by any reason already uploaded file parts (chunks) stay on the storage to be able to resume the file upload in a following upload run. Resuming an upload is only possible until the partial upload is expired and deleted, respectively. To cleanup chunks (expire and delete) originating from unfinished uploads administrators can use this newly introduced command. The expiry time default is two days but can be specified as a parameter of this command.
+**It is recommended to configure CRON to execute this background job regularly (it is not included in the regular ownCloud background jobs to give administrators more flexibility in scheduling)**. Please check the ownCloud documentation for more information. **LINK (https://github.com/owncloud/documentation/pull/3522)**
```

Comment:
As above, enable the link directly in the text:

---

**It is recommended to configure Cron to execute this background job regularly.** It is not included in the regular ownCloud background jobs so that the administrators have more flexibility in scheduling it. Please check [the ownCloud documentation](https://github.com/owncloud/documentation/pull/3522) for more information.

Reply:
Cron --> CRON (preferred) or ``cron``

Reply:
changed